app_server/app/adapters/socket/Room.js

```diff
@@ -75,4 +75,21 @@ Class.removeAllClients = function() {
   this.__clients = {};
 };
 
+Class.getNumberOfUsers = function() {
+  let users = {};
+  let count = 0;
+  for (var socketId in this.getClients()) {
```

Comment:
It's not supposed to work?
I tried it in the console beforehand and it worked, but is it unexpected behaviour?